tests/dummy/app/pods/demo/template.hbs

```diff
@@ -17,7 +17,7 @@
       &lt;placeholder: controls&gt;
   {{/block-slot}}
   {{#block-slot 'actions' as |action|}}
-    {{action.button icon='frost/infobar-create' text='Create' onClick=(action 'createUser')}}
+    {{action.button icon='frost/infobar-create' text='Click me!' onClick=(action 'triggerAction')}}
```

Comment:
We should also make a push to eliminate references to nested icon paths - i.e. `frost/infobar-create` should be `infobar-create` - although I'm curious why we have a second icon for a `plus`